src/vscode-bicep/src/commands/deploy.ts

```diff
@@ -511,11 +519,20 @@ export class DeployCommand implements Command {
         parameterFilePath = result.data;
       }
 
-      if (await this.validateIsValidParameterFile(parameterFilePath, true)) {
+      if (parameterFilePath.endsWith(".bicepparam")) {
         this.outputChannelManager.appendToOutputChannel(
-          `Parameter file used in deployment -> ${parameterFilePath}`
+          `Bicep Parameter file used in deployment -> ${parameterFilePath}`
         );
 
+        return parameterFilePath;
+      }
+      else {
+        if (await this.validateIsValidParameterFile(parameterFilePath, true)) {
```

Comment:
Rather than introducing more conditions here, could we update `validateIsValidParameterFile` to also validate for `.bicepparam`?